test/instance-method.js

```diff
@@ -12,7 +12,7 @@ function createModernFixtures() {
       this.state = { counter: 0 };
     }
 
-    increment() {
+    increment(a) {
```

Comment:
In my opinion it would be better to test this code in separated unit tests instead of incorporating it into existing ones. It may confuse people in the future why the `a` is needed here.